app/analytics.go

```diff
@@ -40,6 +40,7 @@ func (a *App) GetAnalytics(name string, teamId string) (model.AnalyticsRows, *mo
 		rows[8] = &model.AnalyticsRow{Name: "daily_active_users", Value: 0}
 		rows[9] = &model.AnalyticsRow{Name: "monthly_active_users", Value: 0}
 		rows[10] = &model.AnalyticsRow{Name: "inactive_user_count", Value: 0}
+		rows[11] = &model.AnalyticsRow{Name: "registered_users", Value: 0}
```

Comment:
Feels like a dup, isn't this already on the admin console page?

Reply:
@coreyhulen , when we send the telemetry, the "registered_users"/usercount IN TRACK_ACTIVITY count maps to :
```if count, err := a.Srv().Store.User().Count(model.UserCountOptions{IncludeDeleted: true}); err == nil {
		userCount = count
	}``` 
That is the definition I used, and I didn't see it the same for the admin console page display, but it could we get to the same number through some other query. I'll double-check.

Reply:
Hmm, it should be the metric on the system stats named `Total Active Users`. That's what we use to report for E10/E20 licenses.  It doesn't count deleted (I believe) and that's OK.

Reply:
@coreyhulen , to be clear, the "registered_users" metric value here is extracted to be displayed in the message we send to acknowledge@.., not the one we use to determine to send/not-send the warning. 
In https://www.notion.so/Nag-ware-when-over-100-users-008505849eae45a98ae29e53a32b296a , we wanted to display: "Registered Users: XXXX", so I used the "registered_users" definition from the telemetry (the one with {`IncludeDeleted: true`}). 
I'll revert this part and use the `unique_user_count` value for the "Registered Users: XXXX" but then maybe we should then either 
1) change the text in the mail from "Registered Users XXXX" to smth. like "Active Users XXX" or 
2) change the "registered_users" from TRACK_ACTIVITY to smth. else (like `userCount` maybe).

Reply:
removed the dup and reusing the active user count definition as surfaced in the SysConsole.